chapters/VK_EXT_debug_utils.txt

```diff
@@ -42,7 +42,11 @@ fname:vkSetDebugUtilsObjectNameEXT as defined below.
 --
 include::{generated}/api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
-  * pname:device is the device that created the object.
+  * pname:device is the device that created the object, or, if the device is a
+    slink:VkInstance or a slink:VkPhysicalDevice, a device that was created from the object.
+    If the object does not have any relationship to a slink:VkDevice (for example a
+    slink:VkSurfaceKHR), it is a device that was created from the same slink:VkInstance as
+    the object.
```

Comment:
Perhaps the following sentence might be better...

```
* pname:device is the name of a valid slink:VkDevice which is related to the object in
  pname:pNameInfo.
  If pname:pNameInfo references an instance-level object (for example a
  slink:VkInstance or slink:VkPhysicalDevice), pname:device must be created from that object.
  Otherwise, the object passed in pname:pNameInfo must have been created from
  the same slink:VkDevice ancestor as listed in pname:device.
```
Or something like that.  I don't know if there's too much in that statement as well.

Reply:
Good idea, this would be clearer than what I have at the moment. Your statement about instance-level objects might be problematic though, because there are some instance-level objects one can't create a device from (for example `VkSurfaceKHR`). I would like to split these instance-level objects from `VkInstance` and `VkPhysicalDevice`. Maybe
```
* pname:device is the name of a valid slink:VkDevice which is related to the object in
  pname:pNameInfo.
  If pname:pNameInfo references a slink:VkInstance or a slink:VkPhysicalDevice, 
  pname:device must be created from that object.
  If pname:pNameInfo references any other instance-level object, that object and
  pname:device must be created from the same slink:VkInstance.
  Otherwise, the object passed in pname:pNameInfo must have been created from
  the same slink:VkDevice ancestor as listed in pname:device.
```
would work?

Reply:
That sounds great to me.  Sorry for the delayed response.

Reply:
Done.